cmd/auth-provider-gcp/app/getcredentials_test.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package app
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestValidateAuthFlow(t *testing.T) {
+	type FlagResult struct {
+		Flow  string
+		Error error
+	}
+	tests := []FlagResult{
+		{Flow: gcrAuthFlow, Error: nil},
+		{Flow: dockerConfigAuthFlow, Error: nil},
+		{Flow: dockerConfigURLAuthFlow, Error: nil},
```

Comment:
nit: don't need to specify `Error: nil` since nil is the zero value for the field. It's not wrong to keep it, but you'll see a lot of "go idiomatic" code omit it.

Reply:
Gotcha, done